proxy/m2_actions.c

```diff
@@ -1192,6 +1192,16 @@ action_m2_container_propdel (struct req_args_s *args, struct json_object *jargs)
 	return action_sqlx_propdel(args, jargs);
 }
 
+static char *
+_get_property(char **properties, char *key)
+{
+	for (gchar **p=properties; *p && *(p+1) ;p+=2) {
```

Comment:
If generalizing this function, a check on the non-nullity of `properties` is necessary.